code4lib/2021.code4lib.org
# Switch from Whova Widgets to Jekyll Data Files

This PR:
  - Adds workshop data in `_posts`
  - Adds initial set of schedule files in `schedule/day-*/`
    - **Note** - Until the talk schedule is available, `_includes/presentation_timeline.html` has been disabled.
  - Adds initial set of speaker image files from Whova and random `assets/img/nopics/nopic[0-9].png` files.
  - Applies initial set of style changes

To Test:
  - Ensure outline of conference schedule appears at http://127.0.0.1:4000/schedule/
    - Click around, makes sure links point to each day's schedule (to be added later) 
  - Ensure workshops are at http://127.0.0.1:4000/workshops/
    -  Click around, make sure links point to the proper speaker and workshop

**Note**
We will likely need to address a few contrast issues. Feel free to open a fresh issue for those as you find them.

Close #128
